src/components/Header.tsx

```diff
@@ -1,14 +1,18 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useContext } from 'react';
+
+import { ThemeContext, ThemeContextType } from './ThemeContext'
+
 import ThemeToggler from './ThemeToggle';
+import SearchField from './SearchField';
 
 import '../styles/Header.scss';
-import SearchField from './SearchField';
```

Comment:
Maybe it's better to lift this import up and place it above styles import

Reply:
this import is placed above, red line means it deleted from the file :)

Reply:
You are so right. My bad :)